client/my-sites/exporter/index.jsx

```diff
@@ -1,62 +1,47 @@
 /**
  * External dependencies
  */
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import flowRight from 'lodash/flowRight';
 
 /**
  * Internal dependencies
  */
-import analytics from 'lib/analytics';
 import config from 'config';
-import Exporter from './exporter';
-import { getSiteSlug } from 'state/sites/selectors';
-import {
-	shouldShowProgress,
-	getSelectedPostType,
-	isExporting,
-	getExportingState,
-} from 'state/site-settings/exporter/selectors';
-import {
-	advancedSettingsFetch,
-	exportStatusFetch,
-	setPostType,
-	startExport,
-} from 'state/site-settings/exporter/actions';
-import { States } from 'state/site-settings/exporter/constants';
+import { getSelectedSiteId } from 'state/ui/selectors';
 
-function mapStateToProps( state ) {
-	const siteId = state.ui.selectedSiteId;
+import Notices from './notices';
+import ExportCard from './export-card';
+import GuidedTransferCard from './guided-transfer-card';
+import InProgressCard from './guided-transfer-card/in-progress';
+
+class Exporter extends Component {
+	render() {
+		const {
+			siteId,
+			isGuidedTransferInProgress,
+		} = this.props;
+		const showGuidedTransferOptions = config.isEnabled( 'manage/export/guided-transfer' );
 
+		return <div className="exporter">
+			<Notices />
+			{ showGuidedTransferOptions && isGuidedTransferInProgress &&
+				<InProgressCard /> }
+			<ExportCard siteId={ siteId } />
+			{ showGuidedTransferOptions && ! isGuidedTransferInProgress &&
+				<GuidedTransferCard /> }
+		</div>;
```

Comment:
Can we use parentheses to move `<div>` onto a new line and keep the indentation consistent?

```js
return (
	<div className="exporter">
		…
	</div>
);
```